File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCloudTree.java
package app.documents.core.network.manager.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.explorer.Explorer;

public class ResponseCloudTree extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private List<Explorer> response;

    public List<Explorer> getResponse() {
        return response;
    }

    public void setResponse(List<Explorer> response) {
        this.response = response;
    }
}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCloudTree.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import app.documents.core.network.manager.models.explorer.Explorer
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseCloudTree(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: List<Explorer>
) : BaseResponse()